### PR TITLE
fix(copy-button): fix type of feedback text

### LINKED ISSUE #4606: (code snippet) multi/single line have incorrect feedback message type token
The feedback message tooltip text should be using `body-short-01` instead of `code-01`

### PR DESCRIPTION
**Pull request by asudoh**
Fixes #4606.

#### Changelog

**Changed**

- Choice of type token for feedback text.

#### Testing / Reviewing

Testing should make sure copy button and code snippet is not broken.

### CODE REVIEW

**Review by emyarod — COMMENTED** [review]
copy button and inline code snippet look correct, but single line and multiline code snippet feedback text still appear to be using a different type token

image

image

**Comment by abbeyhrt** [comment]
I'm seeing the same thing as @emyarod

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Vanilla and React:

-The `code snippet single line` and the `copy button` should 4px between the border of the hover and the tip of the fin of the tooltip. Right now its too close. 
Screen Shot 2019-11-14 at 1 22 25 PM

-Seeing the same thing as andrew and abbey about the type

**Review by joshblack — APPROVED** [review]

**Review by joshblack — COMMENTED** [review]
Seems great with feedback from above 😄

File: packages/components/src/components/code-snippet/_code-snippet.scss [hunk 1]
```diff
@@ -302,9 +302,23 @@
   }
 
   .#{$prefix}--snippet-button .#{$prefix}--btn--copy__feedback {
-    top: rem(25px);
+    top: rem(50.8px);
```
Comment (joshblack) [inline packages/components/src/components/code-snippet/_code-snippet.scss]: Where is this number coming from?

File: packages/components/src/components/code-snippet/_code-snippet.scss [hunk 2]
```diff
@@ -302,9 +302,23 @@
...
+
+    &:after {
+      top: rem(-4px);
+    }
+  }
+
+  .#{$prefix}--snippet--multi
+    .#{$prefix}--snippet-button
+    .#{$prefix}--btn--copy__feedback {
+    top: rem(42.8px);
```
Comment (joshblack) [inline packages/components/src/components/code-snippet/_code-snippet.scss]: Where is this number coming from?

**Review by joshblack — CHANGES_REQUESTED** [review]
Any chance we could avoid the magic numbers or document them? Seems like we're setting up the placement to be very precise with the decimal pixels

**Comment by asudoh** [comment]
Fixed the tooltip offset, and added the comment how the right offset is generated. Made a change to force serif font as our type token of choice doesn't dictate the font face and code snippet UI has monospace font.

**Review by emyarod — COMMENTED** [review]
the magic numbers in code snippet should be resolved by pending work for #4605 right?

on a side note the copy button tooltip positioning should be resolved in #4715

File: packages/components/src/components/code-snippet/_code-snippet.scss [hunk 3]
```diff
@@ -302,9 +303,27 @@
...
 
   .#{$prefix}--snippet-button .#{$prefix}--btn--copy__feedback {
-    top: rem(25px);
+    top: rem(
+      50.8px
+    ); // (The height of button) + (The height of the tooltip's triangle) + 4px
     left: 50%;
     right: auto;
+
+    &:before {
```
Comment (emyarod) [inline packages/components/src/components/code-snippet/_code-snippet.scss]: should we use the CSS3 selector with double colon instead? same with L316

```suggestion
    &::before {
```

**Review by laurenmrice — APPROVED** [review]
placement of tooltip and the type looks good to me 👍

**Review by joshblack — APPROVED** [review]

**Pull request merged**